### PR TITLE
NGSTACK-876 add scss files for notification module

### PR DESCRIPTION
**Pull request by TTrakostanec**

### CODE REVIEW

**Review by Ljudevit — APPROVED** [review]
thanks @TTrakostanec

File: src/Admin/Menu/MainMenuBuilder.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+namespace App\Admin\Menu;
+
+use App\Attribute\AsMenuBuilder;
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use Netgen\Conversations\Participant\ParticipantProviderInterface;
+use Netgen\Conversations\Repository\ConversationRepository;
+
+#[AsMenuBuilder('createMenu', 'app.admin.main_menu')]
```
Comment (emodric) [inline src/Admin/Menu/MainMenuBuilder.php:13]: Again added in error? Or wrong rebase?

File: package.json [hunk 2]
```diff
@@ -73,9 +73,11 @@
         "@netgen/javascript-cookie-control": "^0.0.8",
         "@popperjs/core": "^2.11.6",
         "bootstrap": "^5.3.3",
+        "flatpickr": "^4.6.13",
```
Comment (emodric) [inline package.json:76]: Didn't I see these changes in your other PR?

File: src/Admin/Menu/MainMenuBuilder.php (hunk 1 above)
Comment (TTrakostanec) [reply src/Admin/Menu/MainMenuBuilder.php]: Rebase error. Fixed.

File: package.json (hunk 2 above)
Comment (TTrakostanec) [reply package.json]: Yes, rebase error. Fixed.

**Comment by emodric** [comment]
Thanks @TTrakostanec !

**Pull request merged**